scripts/test_python_version.py

```diff
@@ -0,0 +1,39 @@
+#!/bin/env python
+
+import pathlib as pl
+import platform
+import sys
+
+
+def main():
+    major_v, minor_v, patch_v = platform.python_version_tuple()
+
+    print(f"Found python version: {major_v}.{minor_v}.{patch_v}")
+
+    min_major_v, min_minor_v = to_version(
```

Comment:
@pcrespov is this better? Am getting the version from the file in the requirements folder now.